### PR TITLE
html-web-page - Jeff Kwon

### PR DESCRIPTION
**Pull request by Jykwon24**
Screen Shot 2021-10-03 at 10 13 53 AM

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <tr>
+      <th></th>
+      <th scope="col">Basic Plan</th>
+      <th scope="col">Deluxe Plan</th>
+      <th scope="col">Jelly Giant Plan</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <th scope="row">Three new curated jellies per month</th>
```
Comment (Zhiwutian) [inline html-web-page/index.html:40]: Each of these first column elements should be TDs, you don't want the text centered as a heading.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

File: html-web-page/index.html (hunk 1 above)
Comment (Jykwon24) [reply html-web-page/index.html]: Screen Shot 2021-10-04 at 5 24 01 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**